BHoM_Engine/Compute/ExtractAssembly.cs

```diff
@@ -164,7 +164,7 @@ private static void ExtractMethods(Assembly asm)
                         // Get only the BHoM methods
                         if (!type.IsInterface && type.IsAbstract)
                         {
-                            foreach (MethodInfo info in type.GetMethods(bindingBHoM).Where(x => x.IsLegal()))
+                            foreach (MethodInfo info in type.GetMethods(bindingBHoM).Where(x => x.IsLegal() && !x.IsSpecialName))
```

Comment:
Thinking about it, shouldn't the check against `IsSpecialName` be a part of `IsLegal`?

Reply:
Thought about that as well, but had a look at the IsLegal method, and seems that was a method added by myself a long time ago to protect against a complete crash. Not sure if I want to involve the IsSpecialName in that half mess, as ideally, the IsLegal should not even be there/required.

Might be able to be convinced otherwise, but that was my thought process at least.

Reply:
Following the conversation with @adecler and @IsakNaslundBh, let's keep the code as is now 👍